source/lib/cli/stimpak.cli.js

```diff
@@ -17,26 +17,38 @@ switch (firstArgument) {
 			.destination(process.cwd());
 
 		const lastArguments = process.argv.splice(2);
-		const answers = [];
+		const generators = [];
+		const answers = {};
 
 		for (let argumentIndex in lastArguments) {
 			const argument = lastArguments[argumentIndex];
 			if (argument.indexOf("--") !== -1) {
-				answers.push(argument);
-			} else {
-				const generatorName = argument;
-				const packageName = `stimpak-${generatorName}`;
-
-				try {
-					const GeneratorConstructor = require(packageName).default;
-					stimpak.use(GeneratorConstructor);
-				} catch (error) {
-					const errorMessage = `"${generatorName}" is not installed. Use "npm install stimpak-${generatorName} -g"`;
+				const md = /^--([^=]+)=(.*)$/.exec(argument);
```

Comment:
Team rule: avoid abbreviated variable names. use verbose ones instead. 

I will refactor this. Don't worry about it.